### DISCUSSION TITLE
LoongArch Support

### DISCUSSION THREAD

**Discussion by qjasn** [jart/blink#157] [Q&A]
> LoongArch is a new RISC ISA, which is a bit like MIPS or RISC-V.

Can this project run on Loongarch? Or what should I do to make it run on Loongarch?

**Comment by jart** [comment]
I haven't ported it to loongarch myself. It's easy to port Blink to new architectures. Slightly harder if you want it to have JIT support too. Contributions would be welcome for any architecture you need.
